etc/notebooks/sentiment_boilerplate_demo/sentiment_boilerplate_demo.ipynb

```diff
@@ -0,0 +1,797 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "urth": {
+     "dashboard": {
+      "hidden": true
+     }
+    }
+   },
+   "source": [
+    "#Sentiment Boilerplate\n",
+    "\n",
+    "This notebook provides a boilerplate for displaying and aggregating metrics about sentiment into a dashboard. In this notebook sentiment is defined as a positive or negative classification for some metric.\n",
```

Comment:
How about this:

> This notebook is a boilerplate for displaying and aggregating metrics about sentiment into a dashboard. This notebook is considered a boilerplate because it allows you to plug in your own metrics and then immediately use the dashboard. In this notebook sentiment is defined as a positive or negative classification for some metric.